app/controllers/pages_visited_controller.rb

```diff
@@ -7,7 +7,9 @@ def show
 private
```

Comment:
This commit msg also needs some newlines

Reply:
Commit message looks fine here - could you elaborate please?

Reply:
These commits

 https://github.com/alphagov/govuk-user-intent-survey-explorer/pull/32/commits/9a119685fb7d606ba4410996e5fd552e228d8273

https://github.com/alphagov/govuk-user-intent-survey-explorer/pull/32/commits/20560169dd8eb78cfa7ff9d45dbd08d13e40d69c

the gds way says commit messages should be around 72 characters per line, both have the message all in one line (they're good messages, no need to change the content)